src/apytypes_wrapper.cc

```diff
@@ -296,6 +296,32 @@ void bind_common(nb::module_& m)
 
             )pbdoc"
         )
+        .def(
+            "set_preferred_array_lib",
+            &set_preferred_array_lib_from_str,
+            R"pbdoc(
+            The set preferred third-party array library to use with APyTypes.
+
+            Parameters
+            ----------
+            array_lib : Literal["numpy", "pytorch", "tensorflow", "jax", "cupy"]
```

Comment:
```suggestion
            array_lib : { "numpy", "pytorch", "tensorflow", "jax", "cupy" }
```
is the Numpydoc way to write it.